src/js/modules/infragistics.ui.scroll.js

```diff
@@ -2161,42 +2163,72 @@
 				this._switchFromTouchToMixed();
 			}
 
+			/** Get current scroll positions to base the next scroll position from */
+			if (this.options.scrollOnlyHBar) {
+				this._startX = this._getScrollbarHPosition();
+			} else {
+				this._startX = this._getContentPositionX();
+			}
+			if (this.options.scrollOnlyVBar) {
+				this._startY = this._getScrollbarVPosition();
+			} else {
+				this._startY = this._getContentPositionY();
+			}
+
+			/** Get delta for the X axis*/
+			if (evt.wheelDeltaX) {
+				/* Option supported on Chrome, Safari, Opera.
+				/* 120 is default for mousewheel on these browsers. Other values are for trackpads */
+				scrollDeltaX = -evt.wheelDeltaX / 120;
```

Comment:
@skrustev Move 120 into a variable or constant

Reply:
There is a comment for that, there's no need for extra variable since extra variable does not describe fully what it is.